sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2244,6 +2244,15 @@ object GpuOverrides {
         ("map", TypeSig.MAP.nested(TypeSig.STRING), TypeSig.MAP.nested(TypeSig.all)),
         ("key", TypeSig.lit(TypeEnum.STRING), TypeSig.all)),
       (in, conf, p, r) => new GpuGetMapValueMeta(in, conf, p, r)),
+    expr[ElementAt](
+      "Returns element of array at given(1-based) index in value if column is array. " +
+        "Returns value for the given key in value if column is map.",
+      ExprChecks.binaryProjectNotLambda(
+        TypeSig.commonCudfTypes, TypeSig.all,
+        ("left", TypeSig.ARRAY.nested(TypeSig.commonCudfTypes) +
```

Comment:
I am not sure that left/right are the right names to use for the parameters. I think it would be clearer to use something like `array/map` and `index/key` for it, which is what uses for their own documentation.

Reply:
Agree, it's more clear, updated.